_13-ex.Rmd

```diff
@@ -53,10 +52,13 @@ sum(d_intersect$all) / sum(desire_lines$all) # 88%
 ```
 
 E4. The analysis presented in this chapter is designed for teaching how geocomputation methods can be applied to transport research. If you were doing this for real, in government or for a transport consultancy, what top 3 things would you do differently?
-<!-- Higher level of geographic resolution. -->
-<!-- Use cycle-specific routing services. -->
-<!-- Identify key walking routes. -->
-<!-- Include a higher proportion of trips in the analysis -->
+
+```{r}
```

Comment:
Why this?

Reply:
To render it on the solution website. If it is an html comment, it is invisible there. However, if it is an R code comment -- you can see it on the solution website.

Reply:
Nice! Forgot that detail.